pkg/cli/service.go

```diff
@@ -844,17 +844,21 @@ func (cve cveResult) stringYAML() (string, error) {
 type fixedTags struct {
 	Errors []errorGraphQL `json:"errors"`
 	Data   struct {
-		ImageList []imageStruct `json:"ImageListWithCVEFixed"` //nolint:tagliatelle // graphQL schema
+		PaginatedImagesResult PaginatedImagesResult `json:"ImageListWithCVEFixed"` //nolint:tagliatelle // graphQL schema
```

Comment:
I noticed in the search tests it's more convenient to embed the PaginatedImagesResult struct. What if we make that change in the ZLI files as well?